modules/installation-configuration-parameters.adoc

```diff
@@ -195,6 +221,185 @@ endif::[]
+
+ifndef::bare[]
+Only IPv4 addresses are supported.
+endif::bare[]
+ifdef::bare[]
+If you use the OVN-Kubernetes cluster network provider, both IPv4 and IPv6 address families are supported.
+
+If you use the OpenShift SDN cluster network provider, only the IPv4 address family is supported.
+
+If you configure your cluster to use both IP address families, you must specify IPv4 and IPv6 addresses in the same order for all network configuration parameters. For example, in the following configuration IPv4 addresses are listed before IPv6 addresses.
+
+[source,yaml]
+----
+networking:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  - cidr: fd00:10:128::/56
+    hostPrefix: 64
+  serviceNetwork:
+  - 172.30.0.0/16
+  - fd00:172:16::/112
+----
+endif::bare[]
+
+.Network parameters
+[cols=".^2,.^3a,.^3a",options="header"]
+|====
+|Parameter|Description|Values
+
+|`networking`
+|The configuration for the network for the cluster.
+|Object
+
+[NOTE]
+====
+You cannot modify parameters specified by the `networking` object after installation.
+====
+
+|`networking.networkType`
+|The cluster network provider Container Network Interface (CNI) plug-in to install.
+|
+ifdef::openshift-origin[]
+Either `OpenShiftSDN` or `OVNKubernetes`. The default value is `OVNKubernetes`.
+endif::openshift-origin[]
+ifndef::openshift-origin[]
+Either `OpenShiftSDN` or `OVNKubernetes`. The default value is `OpenShiftSDN`.
+endif::openshift-origin[]
+
+|`networking.clusterNetwork`
+|
+The IP address blocks for pods.
+
+The default is `10.128.0.0/14` with a host prefix of `/23`.
+
+If you specify multiple IP address blocks, the blocks must not overlap.
+|An array of objects. For example:
+
+[source,yaml]
+----
+ifndef::bare[]
+networking:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+endif::bare[]
+ifdef::bare[]
+networking:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  - cidr: fd01::/48
+    hostPrefix: 64
+endif::bare[]
+----
+
+|`networking.clusterNetwork.cidr`
+|
+Required if you use `networking.clusterNetwork`. An IP address block.
+
+ifndef::bare[]
+An IPv4 network.
+endif::bare[]
+ifdef::bare[]
+If you use the OpenShift SDN network provider, specify an IPv4 network. If you use the OVN-Kubernetes, you can specify either an IPv4 or IPv6 network.
+endif::bare[]
+|
+An IP address block in Classless Inter-Domain Routing (CIDR) notation.
+The prefix length for an IPv4 block is between `0` and `32`. For example, `10.128.0.0/14`.
+ifdef::bare[]
+The prefix length for an IPv6 block is between `0` and `128`. For example, `10.128.0.0/14` or `fd01::/48`.
+endif::bare[]
+
+|`networking.clusterNetwork.hostPrefix`
+|The subnet prefix length to assign to each individual node. For example, if `hostPrefix` is set to `23`, then each node is assigned a `/23` subnet out of the given `cidr`, allowing for 510 (2^(32 - 23) - 2) pod IP addresses.
```

Comment:
Disregard if this has an original someplace else:

* Remove the then in if/then.  (+1 for me for an ISG find.)

* The "For example" sentence seems like a run-on.  As an obligatory sugg, maybe "...out of the given `cidr`. A `hostPrefix` value of `23` provides 510 (2^(32 - 23) - 2) IP addresses for pods.